module.c

```diff
@@ -442,6 +509,77 @@ static PyObject *context_gc(ContextData *self) {
 	Py_RETURN_NONE;
 }
 
+static JSValue js_c_function(
+    JSContext *ctx, JSValueConst this_val, int argc, JSValueConst *argv, int magic) {
+	ContextData *self = (ContextData *)JS_GetContextOpaque(ctx);
+	PythonCallableNode *node = self->python_callables;
+	while (node && node->magic != magic) {
```

Comment:
This loop is slow when large chain.

Reply:
I think this will not be a problem in practice though?